src/audio/drc/drc_math.h

```diff
@@ -18,7 +18,7 @@
 /* Unmark this define to use cordic arc sine implementation. */
 /* #define DRC_USE_CORDIC_ASIN */
 
-#if SOF_USE_HIFI(4, DRC) || SOF_USE_HIFI(3, DRC)
+#if SOF_USE_HIFI(5, DRC) || SOF_USE_HIFI(4, DRC) || SOF_USE_HIFI(3, DRC)
```

Comment:
Yup, I agree with Curtis that this seems a bit silly and we'll have same problem with HIFI6/7. I think this macro continues to be valid for code that targets a specific hifi version, but given it seems we have code that should be compiled with whatever max hifi level, I think something like:

```
#define SOF_MIN_HIFI(level, component) (SOF_CONFIG_HIFI(level, component) || \
»       (SOF_CONFIG_HIFI(MAX, component) && level <= SOF_MAX_XCHAL_HIFI))
```

and then use "SOF_MIN_HIFI(3, DRC)" in this code. Can you @singalsu test this out?
specifically uses